defaults/main.yml

```diff
@@ -1,18 +1,50 @@
 ---
 ## General
-airflow_version: 1.9.0
+airflow_version: 1.10.12
 # Available extra packages: https://airflow.apache.org/installation.html#extra-packages
 # List should follow Ansible's YAML basics: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics
 airflow_extra_packages: []
 
-airflow_pip_executable: "pip"
-airflow_executable: "/usr/local/bin/airflow"
+airflow_python_download: "https://www.python.org/ftp/python/3.8.6/Python-3.8.6.tgz"
+
+airflow_pip_executable: "{{ airflow_home }}/python/bin/pip3"
+airflow_pip_venv_executable: "{{ airflow_home }}/airflow_venv/bin/pip3"
+airflow_executable: "{{ airflow_home }}/airflow_venv/bin/airflow"
 
 airflow_required_libs:
```

Comment:
It would be nice to add this (http://airflow.apache.org/docs/stable/installation.html) as comment before that list :)